protocol-impl/src/main/java/io/zeebe/protocol/impl/record/value/deployment/DeploymentDistributionRecord.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.protocol.impl.record.value.deployment;
+
+import io.zeebe.msgpack.property.IntegerProperty;
+import io.zeebe.protocol.impl.record.UnifiedRecordValue;
+import io.zeebe.protocol.record.value.DeploymentDistributionRecordValue;
+
+public class DeploymentDistributionRecord extends UnifiedRecordValue
+    implements DeploymentDistributionRecordValue {
+
+  private final IntegerProperty partitionProperty = new IntegerProperty("partition");
```

Comment:
(optional) We can make it a bit more clear that the property holds the id of the partition. PartitionId is a common term.

```suggestion
  private final IntegerProperty partitionIdProperty = new IntegerProperty("partitionId");
```